### PR TITLE
Update foreman-tasks up to 4.0.0

### PR DESCRIPTION
**Pull request by adamruzicka**
- Update foreman-tasks to 3.0.4
- Update foreman-tasks to 3.0.5
- Update ruby-foreman-tasks to 4.0.0

### CODE REVIEW

File: plugins/ruby-foreman-tasks/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,21 @@
+ruby-foreman-tasks (3.0.5-1) stable; urgency=low
```
Comment (evgeni) [inline plugins/ruby-foreman-tasks/debian/changelog:1]:
```suggestion
ruby-foreman-tasks (4.0.0-1) stable; urgency=low
```

File: plugins/ruby-foreman-tasks/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+  * 4.0.0 released
+
+ -- Adam Ruzicka <aruzicka@redhat.com>  Tue, 02 Mar 2021 14:06:47 +0100
+
+ruby-foreman-tasks (4.0.0-1) stable; urgency=low
```
Comment (evgeni) [inline plugins/ruby-foreman-tasks/debian/changelog:7]:
```suggestion
ruby-foreman-tasks (3.0.5-1) stable; urgency=low
```

**Review by evgeni — CHANGES_REQUESTED** [review]

**Comment by evgeni** [comment]
damn, this was also broken in `deb/develop` and I didn't notice

**Comment by adamruzicka** [comment]
Soo, should I fix it in develop first and then cp it?

**Comment by evgeni** [comment]
develop fix in https://github.com/theforeman/foreman-packaging/pull/6518

**Review by evgeni — APPROVED** [review]

**Pull request merged**